ABOUT.md

```diff
@@ -1,9 +1,106 @@
 
-Originally focussing on large-scale kernel methods and bioinformatics (for a list of scientific papers mentioning Shogun, see [here](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=)), the toolbox saw massive extensions to other fields in recent years. It now offers features that span the whole space of Machine Learning methods, including many classical methods in classification, regression, dimensionality reduction, clustering, but also more advanced algorithm classes such as metric, multi-task, structured output, and online learning, as well as feature hashing, ensemble methods, and optimization, just to name a few. Shogun in addition contains a number of exclusive state-of-the art algorithms such as a wealth of efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing, Krylov methods, etc. All algorithms are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation & I/O, etc; the resulting combinatorial possibilities are huge. See our [showroom](http://shogun.ml/notebook/latest/).
+#### Shogun is accessible
 
-The wealth of ML open-source software allows us to offer bindings to other sophisticated libraries including: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+* Supports *many languages* (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and *platforms* (Linux/Unix, MacOS and Windows), easy integration with other ML libraries
+
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
+* easy combination of *multiple data representations, algorithm classes and general purpose tools* for rapid prototyping of data pipelines
+
+#### Shogun is open source
+
+* *free* and *community-based* machine learning education (see [Using Shogun to Learn](#useshoguntolearn))
+* *GPLv3 License* and working towards BSD compatibility
```

Comment:
What about wording this so that it reads that BSD license (we could perhaps add being worked on), although a few parts rely on GPLv3.